react/features/room-lock/components/RoomLockButton.js

```diff
@@ -43,7 +50,17 @@ class RoomLockButton extends AbstractButton<Props, *> {
      * @returns {void}
      */
     _handleClick() {
-        this.props.dispatch(beginRoomLockRequest());
+        const { _lockUnlockEnabled } = this.props;
+
+        if (_lockUnlockEnabled) {
+            const { dispatch, _locked } = this.props;
+
+            if (_locked) {
+                dispatch(unlockRoom());
```

Comment:
The new UI we want for this is a new dialog. I know you were not aware of this, but in a nutshell we need a confirmation dialog: "Are you sure you want to unlock the room?" with options of Ok and Cancel.

@lyubomir Do you feel like we should do it in a single go? The dialog can be added later on of course.

Reply:
Is the new dialog going to take a lot of time to develop, review, & merge? This PR is supposed to fix a bug which is kind of more urgent than a "feature request" which is a new dialog UI. But if the difference in development time is going to be at most 1 day, I don't mind waiting for the dialog.